config/configgrpc/configgrpc_test.go

```diff
@@ -102,11 +102,10 @@ func TestDefaultGrpcClientSettings(t *testing.T) {
 	tt, err := componenttest.SetupTelemetry(componentID)
 	require.NoError(t, err)
 	t.Cleanup(func() { require.NoError(t, tt.Shutdown(context.Background())) })
-
+	clientConfig := configtls.NewDefaultClientConfig()
```

Comment:
nit: The term "clientConfig" is highly overloaded in this context, I wonder if it would be better to rename this local "tlsSettings" or something similar.